src/Fable.Core/Fable.Core.PY.fs

```diff
@@ -2,6 +2,7 @@ namespace Fable.Core
 
 open System
 
+[<RequireQualifiedAccess>]
 module PY =
```

Comment:
Do you need to rename it to Py? or is PY fine. I actually would prefer `Py`. Then the file should also be renamed.

Reply:
Either will work. If you prefer `Py` probably it's a good opportunity to rename the module 👍

Reply:
Yes please, JS is JavaScript. Python should be Py

Reply:
Ok, [I've changed PY to Py](https://github.com/fable-compiler/Fable/pull/3185/commits/2971344f593df3fafde1a7ac32f7acffa081bbde). Doing a search & replace I also hit `Fable.PY` module within Fable code itself. Did you want to change that as well or only in Fable.Core?

Reply:
Great. I can fix `Fable.PY` later since it's not exposed to the user. Feel free to merge 👍